core/templates/dev/head/pages/landing-pages/stewards-landing-page/stewards-landing-page.mainpage.html

```diff
@@ -1,22 +1,269 @@
+      .oppia-stewards-button-secondary:hover,
+      .oppia-stewards-button-secondary:focus,
+      .oppia-stewards-button-secondary:active {
+        border-color: #05beb2;
+        color: #05beb2;
+      }
+      .oppia-stewards-main-content {
+        float: left;
+        height: 500px;
+        margin-left: 20px;
+      }
+      .oppia-stewards-main-content-buttons {
+        display: block;
+        margin: 40px auto 0;
+        text-align: center;
+      }
+      .oppia-stewards-landing-top-section {
+        margin-left: 60px;
+      }
+      .oppia-stewards-landing-background-icon-row {
+        margin-top: 0;
+        margin-bottom: 0;
+        margin-left: -webkit-calc((100vw - 2700px) / 2);
+        margin-left: -moz-calc((100vw - 2700px) / 2);
+        margin-left: -o-calc((100vw - 2700px) / 2);
+        margin-left: calc((100vw - 2700px) / 2);
+        margin-right: -webkit-calc((100vw - 2700px) / 2);
+        margin-right: -moz-calc((100vw - 2700px) / 2);
+        margin-right: -o-calc((100vw - 2700px) / 2);
+        margin-right: calc((100vw - 2700px) / 2);
+        opacity: 0.4;
+        position: relative;
+        text-align: center;
+        width: 2700px;
+      }
+      .oppia-stewards-landing-background-icon {
+        margin: -1px;
+        max-width: 96px;
+        width: 10%;
+      }
+
+      /* Note: This should be kept in sync with the CSS media query on
+          stewards-landing-page.mainpage.html.
+      */
+      @media screen and (max-width: 890px) {
+        .oppia-stewards-landing-background-icon-row-container {
+          max-width: 100vw;
+          overflow: hidden;
+          margin-left: -54px;
+        }
+        .oppia-stewards-landing-section-inner,
+        .oppia-stewards-landing-section {
+          height: auto;
+        }
+        .oppia-stewards-landing-section-inner {
+          margin-left: 54px;
+        }
+        .oppia-stewards-landing-h1 {
+          font-size: 1.6em;
+          line-height: 1.6em;
+          margin-right: 20%;
+          padding-left: 0;
+          padding-right: 0;
+          padding-top: 40px;
+          width: 80%;
+        }
+        .oppia-stewards-landing-h2 {
+          font-size: 1em;
+          width: 80%;
+        }
+        .oppia-stewards-landing-top-section {
+          margin-left: auto;
+        }
+        .oppia-stewards-button {
+          /* This font size is used in oppia.css for the button in splash page in
+              mobile view. */
+          font-size: 3.6vw;
+          margin-bottom: 30px;
+          width: auto;
+        }
+        .oppia-stewards-main-content {
+          width: 90%;
+        }
+      }
+      @media screen and (max-width: 320px) {
+        .oppia-stewards-button {
+          /* This font size is used in oppia.css for the button in splash page in
+              mobile view. */
+          font-size: 3.6vw;
+        }
+        .oppia-stewards-landing-h1 {
+          white-space: normal;
+        }
+      }
+    </style>
```

Comment:
The `<style>` part is already in [stewards-landing-page.directive.html](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/pages/landing-pages/stewards-landing-page/stewards-landing-page.directive.html)